mobile/test/cc/integration/send_headers_test.cc

```diff
@@ -33,28 +33,28 @@ TEST(SendHeadersTest, Success) {
   EngineWithTestServer engine_with_test_server(engine_builder, TestServerType::HTTP2_WITH_TLS);
   engine_running.WaitForNotification();
 
-  int actual_status_code;
+  std::string actual_status_code;
```

Comment:
why is the status code a string instead of an int?  for HTTP response codes, wouldn't int make more sense?

Reply:
That's what `Http::ResponseHeaderMap` (the built-in Envoy data type) provides. I don't think there's an API to get the integer version unless we convert it ourselves. Having less wrappers is better IMO.